src/Models/GameModel.php

```diff
@@ -40,4 +40,41 @@ public function getAllGames(): array
 
         return $games;
     }
+    public function getAllGenres()
```

Comment:
Strictly speaking, you should really put this method into a GenreModel.

When a method in the GameModel interacts with both the game table, and genre table (like getting all games with their genres), its fine to go in here. But because this particular query is only dealing with the genre table, it should go into a new GenreModel